backend/config/settings.py

```
@@ -25,17 +25,24 @@ def load_dotenv(dotenv_path: Path) -> None:
-    print("DEBUG - REDIS_URL prefix:", os.environ["REDIS_URL"][:15])
-
-SECRET_KEY = os.getenv(
-    "SECRET_KEY", "django-insecure-dev-key-not-for-production-use-32bytes!!"
-)
+# SECURITY: Require SECRET_KEY in production. Only use insecure fallback in development.
+SECRET_KEY = os.getenv("SECRET_KEY")
+if not SECRET_KEY:
+    import warnings
+    if DEBUG:
+        # Only warn in debug mode - use an obviously fake key that won't work in production
+        warnings.warn(
```

Ruff (0.15.20)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)